File: DAY1/part2.py
cx = 0
sa = []
fin = open("input.txt", "r")
fin = [int(x) for x in fin.read().split()]
for x in range(len(fin)-2):
    sa.append((fin[x]+fin[x+1]+fin[x+2]))

for x in range(1, len(sa)):
    if sa[x] > sa[x-1]:
        cx += 1

print(cx)

File: DAY2/part2.c
#include <stdio.h>
int main(void) {
	FILE *input;
	int i, x = 0, y = 0, a = 0;
	char buff[255];
	input = fopen("input.txt", "r");
	while (fscanf(input, "%s %d", &buff, &i)) {
		if (feof(input)) {
			break;
		}
		//printf("%s, %d\n", buff, i);
		if (buff[0] == 'f') {
			x += i;
			y += a*i;
		}
		if (buff[0] == 'd') {
			a += i;
		}
		if (buff[0] == 'u') {
			a -= i;
		}
	}
	//Don't really need a lld but just to be safe since my answer was near
	//the maximum range of int
	printf("%lld\n", x*y);
	return 0;
}
			
			

